resources/views/chirps/index.blade.php

```diff
@@ -18,7 +18,7 @@ class="block w-full border-gray-300 focus:border-indigo-300 focus:ring-indigo-20
 		<div class="mt-6 bg-white shadow-sm rounded-lg divide-y">
            		 @foreach ($chirps as $chirp)
                			 <div class="p-6 flex space-x-2">
-								<img src="{{ $chirp->user->profile_image }}"  alt="Current Profile Picture" class="h-16 w-16 object-contain rounded-full"/>
+								<img src="{{ $chirp->user->profile_image }}"  alt="Current Profile Picture" class="h-16 w-16 object-cover rounded-full"/>
```

Comment:
Add a default image if there is no image or if there is an error loading the image